### DISCUSSION TITLE
[FABRIC 1.18.2] Severe FPS drop when opening GUIs with REI installed

### DISCUSSION THREAD

**Discussion by Shadowevil015** [shedaniel/RoughlyEnoughItems#979] [Q & A] [answered]
Hi, I'm currently running a custom modpack for Fabric 1.18.2, which works completely fine with REI, but I've noticed that I keep getting very bad FPS drops when opening my inventory, opening chests, or anything that shows the REI screen. 

The strange thing is, my friends do not get this issue, or at least not to a severe extent, and the only realistic difference would be that i'm running linux, while they're on windows

Attatched is the logs, and also a [spark report](https://spark.lucko.me/cwCvtmwEZX)

[latest.log](https://github.com/shedaniel/RoughlyEnoughItems/files/9077903/latest.log)
[rei-issues.log](https://github.com/shedaniel/RoughlyEnoughItems/files/9077904/rei-issues.log)
[rei.log](https://github.com/shedaniel/RoughlyEnoughItems/files/9077905/rei.log)

If you need anything else, don't hesitate to ask!

Thanks!

**Comment by shedaniel** [comment, marked as answer]
It looks like from your logs that REI Fast Entry Rendering is disabled. Can you confirm that in Config > Advanced > Misc
